frontend/src/metabase/hooks/use-sequenced-content-close-handler.ts

```diff
@@ -75,11 +103,15 @@ export default function useSequencedContentCloseHandler() {
   }, [handleEvent]);
 
   const setupCloseHandler = useCallback(
-    (contentEl: Element | null, close: () => void) => {
+    (instance: TippyInstance, close: () => void) => {
       removeCloseHandler();
 
-      if (isElement(contentEl)) {
-        const popover = { contentEl, close };
+      if (isElement(instance.popper)) {
+        const popover = {
+          triggerEl: instance.reference,
```

Comment:
This will be used to decide if we want to let `click` event propagate up to the trigger's click listener.